README.md

```diff
@@ -63,14 +63,16 @@ See [MG-0](./MG/mg-0) for an example to get started. A template is provided at [
 
 ## Files and numbering
 
-Each MIP, MD or MG is stored in a separate subdirectory with a name `mip-<number>`, `md-<number>` or `mg-<number>`. The subdirectory contains a `README.md` that describes the MIP, MD, or MG. All assets related to the MIP, MD or MG are stored in the same subdirectory.
+An MIP/MD uses the PR number as the MIP/MD number. Note that PRs that do not introduce a new MIP/MD are also accepted. Thus, there will be gaps in the MIP/MD number sequence.
```

Comment:
i just 
- reordered these paras
- the sentences were not clear, so i slightly rephrased
- there was a sentence that was not helpful so removed